src/erlfmt.erl

```diff
@@ -59,7 +59,11 @@ init(State) ->
 
 %% API entry point
 -spec format_file(file:name_all() | stdin, out(), config()) ->
-    {ok, [error_info()]} | skip | {error, error_info()}.
+    {ok, [error_info()]} |
+    {check_failed, string(), string(), [error_info()]} |
+    {[error_info()]} |
```

Comment:
In what case do we return just the bare warnings?

Reply:
Oh that is a typo good catch